doc/doxygen/src/Examples.dox

```diff
@@ -278,8 +281,13 @@
  * # publishing) and can be provided up-front.
  * file_spec = TextFileSpecification.create()
  * file_spec.markupTrait().setMarkupType("plain")
+ *
+ * # We're not, as far as we know, creating a new related entity. We are
+ * # most likely mutating an existing entity. So we use kWrite access
+ * # mode.
```

Comment:
[Consider] Weird take, but to me this sentance sort of gives the vibe that the host should/wants to know about the method of storage on the back end. I wonder if a better phrasing might be

"Our intent is to write data to this entity, not to create a new related entity (kCreateRelated,) so we use the kWrite access mode."

Reply:
Like it. Used your wording in d13f14ca